composer.json

```diff
@@ -11,24 +11,29 @@
     "autoload": {
         "psr-4": {
             "Bref\\SymfonyBridge\\": "src/"
-        }
+        },
+        "files": [
+            "src/bootstrap.php"
```

Comment:
This file registers the Symfony container in Bref. It will be called for every invocation/request.